moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -74,10 +75,11 @@ class Servo
 
   /**
    * \brief Computes the joint state required to follow the given command.
+   * @param a RobotStatePtr instance used for calculating the joint state.
```

Comment:
The second thing in these has to be the actual variable name

```suggestion
   * @param robot_state a RobotStatePtr instance used for calculating the joint state.